.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - checkout
       - run:
           name: Build test binaries
-          command: make build-linux
+          command: make build-linux bin/acceptance.linux_amd64
```

Comment:
I removed the acceptance binary from the default linux targets, as you often don't want to compile this into the dockerfile and it adds another binary for compilation in your feedback cycle